pygmt/datasets/samples.py

```diff
@@ -300,7 +297,21 @@ def list_sample_data():
     return {name: dataset.description for name, dataset in datasets.items()}
 
 
-def load_sample_data(name):
+def load_sample_data(
+    name: Literal[
+        "bathymetry",
+        "earth_relief_holes",
+        "fractures",
+        "hotspots",
+        "japan_quakes",
+        "mars_shape",
+        "maunaloa_co2",
+        "notre_dame_topography",
+        "ocean_ridge_points",
+        "rock_compositions",
+        "usgs_quakes",
+    ],
+):
```

Comment:
So far in line 313 (currently) or 324 (after this PR):
https://github.com/GenericMappingTools/pygmt/blob/09f24521789ca230ae8b75439654ca8c8635b91d/pygmt/datasets/samples.py#L311-L315
was not changed to
```
name
```
Analogous as done for `data_source` in PR #2849.
https://github.com/GenericMappingTools/pygmt/blob/09f24521789ca230ae8b75439654ca8c8635b91d/pygmt/datasets/earth_relief.py#L77-L81
Or was this intended because the list of available input names is a bit longer?

Reply:
Fixed in 10a7ce445867e266038cd64d98d47fc356003115. Now it looks like:

<img width="857" alt="image" src="https://github.com/GenericMappingTools/pygmt/assets/3974108/501620dd-ff85-48e9-85cc-6c48bcd0add9">

Yes, it's a little bit longer, but I feel it's still acceptable.